integration/wrappers_test.go

```diff
@@ -18,12 +18,12 @@ var _ = Describe("Wrappers Integration", func() {
 	Describe("RetrieveAndProcessTables", func() {
 		BeforeEach(func() {
 			rootCmd := &cobra.Command{}
-			includes := []string{"--include-table", "public.foo", "--include-table", "public.BAR"}
-			rootCmd.SetArgs(options.HandleSingleDashes(includes))
 			backup.DoInit(rootCmd) // initialize the ObjectCount
-			rootCmd.Execute()
 		})
 		It("returns the data tables that have names with special characters", func() {
+			_ = backupCmdFlags.Set(options.INCLUDE_RELATION, "public.foo")
+			_ = backupCmdFlags.Set(options.INCLUDE_RELATION, "public.BAR")
```

Comment:
Why did you move it?

Reply:
This is a partial revert of Polina's changes to be closer to the upstream.